Makefile

```diff
@@ -0,0 +1,20 @@
+.DEFAULT_GOAL := build
+.PHONY: build install docker dockerpush
+
+REPO=smartcontractkit/bea-adapter
+LDFLAGS=-ldflags "-X github.com/smartcontractkit/bea-adapter/store.Sha=`git rev-parse HEAD`"
```

Comment:
Does this serve any purpose, as there is no store in this package?

Reply:
Hmm, good one.. I copied it from the linkpool one (that also doesn't have a store it seems). I'll ask Jonny but I'll remove it as well for now. Guess he's using a default Makefile